src/apps/blockScrape/scraper_context.cpp

```diff
@@ -258,38 +258,38 @@ void CScraper::finalizeIndexChunk(void) {
     string_q asciiFile = indexFolder_sorted_v2 + padNum9(first)+"-"+padNum9(last) + ".txt";
     string_q binFile = indexFolder_finalized_v2 + padNum9(first)+"-"+padNum9(last) + ".bin";
 
-    CStringArray apps;
-    apps.reserve(options->maxIndexRows + 100);
+    CStringArray appearances;
+    appearances.reserve(options->maxIndexRows + 100);
 
     cerr << "Processing index.";
     cerr.flush();
 
     for (blknum_t i = first ; i <= last ; i++) {
-        string_q theStuff;
-        string_q fn = indexFolder_staging_v2 + padNum9(i) + ".txt";
-        asciiFileToString(fn, theStuff);
+        string_q stagingList;
+        string_q file = indexFolder_staging_v2 + padNum9(i) + ".txt";
+        asciiFileToString(file, stagingList);
         CStringArray lns;
-        explode(lns, theStuff, '\n');
+        explode(lns, stagingList, '\n');
         for (auto ln : lns) {
-            apps.push_back(ln);
-            if (!(apps.size() % (options->maxIndexRows / 10))) {
+            appearances.push_back(ln);
+            if (!(appearances.size() % (options->maxIndexRows / 10))) {
                 cerr << "."; cerr.flush();
             }
         }
     }
-    sort(apps.begin(), apps.end());
+    sort(appearances.begin(), appearances.end());
 
     size_t cnt = 0;
     ostringstream os;
-    for (auto app : apps) {
+    for (auto app : appearances) {
         os << app << "\n";
         if (!(++cnt % (options->maxIndexRows / 10))) {
             cerr << "."; cerr.flush();
         }
     }
     // TODO(jayrush): should this be stringToAsciiFile not append?
```

Comment:
makes sense. I scanned the file and, other than the declaration, asciiFile is not mentioned or touched -- except here in line 291.